README.md

```diff
@@ -70,6 +70,15 @@ The application logs to ElasticSearch to allow debugging and monitoring. A `stat
 ensure that Elastic chooses the right type for the index fields a dynamic index template must be added to Elastic. This
 can be done with the `app:elastic:create-stats-template` command.
 
+### Low disk
+
+If you have less than 10% disk space left elastic search will not index content and switchs into read-only mode. This
+command will disable this behaviour:
```

Comment:
- Is this really what we want? Thinking there is a reason for ES doing this.
- Do we have alerts set up for low disk space?

Reply:
This is only for lokal development... even when I have 80gb free i do this.